src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ConsoleDebugTargetsProvider.cs

```diff
@@ -178,9 +178,12 @@ private static bool IsRunProjectCommand(ILaunchProfile profile)
             string projectFolder = Path.GetDirectoryName(UnconfiguredProject.FullPath);
             var configuredProject = await GetConfiguredProjectForDebugAsync().ConfigureAwait(false);
 
-            // If no working directory specified in the profile, we default to the one returned from GetRunnableProjectInformationAsync (if running
-            // the project), or the project folder.
-            string defaultWorkingDir = projectFolder;
+            // If no working directory specified in the profile, we default to output directory.
+            string defaultWorkingDir = await GetOutputDirectoryAsync(configuredProject).ConfigureAwait(false);
```

Comment:
Is it possible for OutDir to be unset (manually or otherwise), making it necessary to fallback on project folder?

Reply:
It's definitely possible for it to be manually unset. It'd probably break lots of other things, though.

It might be wise to check for check for empty as well as directory existence.